entrypoint.sh

```diff
@@ -456,7 +456,7 @@ appBackup() {
 }
 appRestore() {
     echo "Starting restore process ..."
-    if [ "$(ls -A "$DATA_DIR/backups/")" ]; then
+    if [ -z "$(ls -A $DATA_DIR/backups)" ]; then
```

Comment:
The inner quotes [should not have been deleted](https://github.com/koalaman/shellcheck/wiki/SC2086).

Reply:
@andersk 	you're right, the inner quotes were fine to leave in